sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -95,11 +95,11 @@ def on_request(self, request):
 
 
 class AzureKeyCredentialPolicy(SansIOHTTPPolicy):
-    """Adds an API key header for signing requests.
+    """Adds a key header for signing requests.
 
     :param credential: The credential used to authenticate requests.
     :type credential: ~azure.core.credentials.AzureKeyCredential
-    :param str key_header: The name of the API key header used for signing requests.
+    :param str key_header: The name of the key header used for signing requests.
```

Comment:
Is "signing" the term other languages are using? For cognitive services, we are not really signing the request with the key (we are only providing the literal value in the request).

Reply:
yeah that's what .NET is doing. @maririos what are your thoughts. "The name of the key header used to provide the credential"?

Reply:
Sometimes a shorter parameter name can help. What about simply calling it `name`? And if we end up having to distinguish between headers and query parameters, when can add a `location` parameter and default it to 'header'.

Reply:
"The name of the key header used for the credential"?